exporter/v1_compatibility.go

```diff
@@ -599,6 +599,16 @@ func makeLockMetric(m bson.M, lm lockMetric) (prometheus.Metric, error) {
 	return prometheus.NewConstMetric(d, prometheus.UntypedValue, *f, lv...)
 }
 
+// PMM dashboards looks for this metric so, in compatibility mode, we must expose it.
+func mongodbUpMetric() prometheus.Metric {
+	d := prometheus.NewDesc("mongodb_up", "Whether MongoDB is up.", nil, nil)
```

Comment:
Shouldnt we check that mongodb is really up?

Reply:
This is being call after successfully collect `getDiagnosticData`, so, the server is up.

Reply:
ok

Reply:
⚠️⚠️⚠️ 

But we really should expose `mongodb_up` as `0` if it is not up, not just skip it. That's important for Prometheus data model.

In fact, we should set `mongodb_up` to `1` or `0` in both modes, right after we connected and pinged database successfully or not.

Please decide if that's should be a separate ticket, but it must be done before release – `mongodb_up` is very important.

/cc @alexanderblinkov-percona

Reply:
@percona-csalguero please create a new ticket for ^ and post it there

Reply:
New ticket created: https://jira.percona.com/browse/PMM-6585